internal/controlplane/handlers_user.go

```diff
@@ -78,7 +80,13 @@ func (s *Server) CreateUser(ctx context.Context,
 		// Set up the default project for the user
 		baseName := subject
 		if token.PreferredUsername() != "" {
-			baseName = token.PreferredUsername()
+			// Check if `project_name_lower_idx` unique constraint was violated. This happens when
```

Comment:
Is this an outdated comment? We check the other way around, to see if there's `sql.ErrNoRows` returned right?

Reply:
You're right, this was a leftover copy-pasta from the other place where we had this check 👍